### DISCUSSION TITLE
Change off-canvas-menu icon

### DISCUSSION THREAD

**Discussion by okmog** [templaza/astroid-framework#751] [Q&A]
Hello,
I'm using Astroid template two. I've put a checkout module in the off-canvas-menu and would like to change change the icon from hamburger to shopping cart. How do I do that? Is it possible by css? TIA

**Comment by okmog** [comment]
Just found a work around by not using the off-canvas-menu for the shopping cart.

**Reply by nidzo80** [reply]
https://github.com/templaza/astroid-framework/discussions/150#discussioncomment-3162481
